src/pages/CalculatorMain.js

```diff
@@ -0,0 +1,10 @@
+import Calculator from '../components/Calculator';
+import '../components/Calculator.css';
+
```

Comment:
- [ ] When I run the application I get the below error, Because the directory name is `/Components` not `/components`. Please fix this typo and make sure the application runs without any errors. 
![image](https://user-images.githubusercontent.com/83803414/189992172-0a7e07f5-9506-47c6-be3b-a55e8ee49747.png)

Reply:
Both the files are part of `components` now. Thanks